src/core/main.cpp

```diff
@@ -778,40 +778,39 @@ int main( int argc, char * * argv )
 		}
 		else
 		{
-
-		// If enabled, open last project if there is one. Else, create
-		// a new one. Also skip recently opened file if limited session to
-		// lower the chance of opening an already opened file.
-		if( ConfigManager::inst()->
-				value( "app", "openlastproject" ).toInt() &&
-			!ConfigManager::inst()->recentlyOpenedProjects().isEmpty() &&
-			gui->mainWindow()->getSession()
-					!= MainWindow::SessionState::Limited )
-		{
-			QString f = ConfigManager::inst()->
-						recentlyOpenedProjects().first();
-			QFileInfo recentFile( f );
-
-			if ( recentFile.exists() )
-			{
-				Engine::getSong()->loadProject( f );
+			// If enabled, open last project if there is one. Else, create
+			// a new one. Also skip recently opened file if limited session to
+			// lower the chance of opening an already opened file.
+			if( ConfigManager::inst()->
+					value( "app", "openlastproject" ).toInt() &&
+				!ConfigManager::inst()->recentlyOpenedProjects().isEmpty() &&
+				gui->mainWindow()->getSession()
+						!= MainWindow::SessionState::Limited )
+			{
+				QString f = ConfigManager::inst()->
+							recentlyOpenedProjects().first();
+				QFileInfo recentFile( f );
+
+				if ( recentFile.exists() )
+				{
+					Engine::getSong()->loadProject( f );
+				}
+				else
+				{
+					Engine::getSong()->createNewProject();
+				}
 			}
 			else
 			{
 				Engine::getSong()->createNewProject();
-			}
-		}
-		else
-		{
-		Engine::getSong()->createNewProject();
-		}
 
-			// [Settel] workaround: showMaximized() doesn't work with
-			// FVWM2 unless the window is already visible -> show() first
-			gui->mainWindow()->show();
-			if( fullscreen )
-			{
-				gui->mainWindow()->showMaximized();
+				// [Settel] workaround: showMaximized() doesn't work with
+				// FVWM2 unless the window is already visible -> show() first
+				gui->mainWindow()->show();
```

Comment:
Is this workaround part being moved into the else clause (a) intentionally (b) by mistake or (c) only in my imagination?